pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -228,6 +229,22 @@ public static long fromEpochDaysBucket(long days, long bucket) {
     return TimeUnit.DAYS.toMillis(days * bucket);
   }
 
+  /**
```

Comment:
There are existing functions in this class like dayOfWeek, dayOfYear etc. I think we should add a version of those on Timestamp type. Alternately, since the stored type is long, may be they will automatically work on column of TIMESTAMP type ?

Reply:
Yes, it automatically works, and that's one reason why TIMESTAMP is stored as millis since epoch